context/app/static/js/components/Detail/ProvAnalysisDetails/ProvAnalysisDetails.jsx

```diff
@@ -1,27 +1,22 @@
-/* eslint-disable react/no-array-index-key */
 import React from 'react';
 import PropTypes from 'prop-types';
-import List from '@material-ui/core/List';
-import Divider from '@material-ui/core/Divider';
 
-import ProvAnalysisDetailsLink from '../ProvAnalysisDetailsLink';
-import { StyledListItem } from './style';
+import ProvAnalysisDetailsList from 'js/components/Detail/ProvAnalysisDetailsList';
 
 function ProvAnalysisDetails(props) {
   const { dagListData } = props;
+
+  const ingestPipelines = dagListData.filter((pipeline) => !('name' in pipeline));
+  const cwlPipelines = dagListData.filter((pipeline) => 'name' in pipeline);
```

Comment:
I'd worry a little bit about data changing upstream and this no longer being a good think to partition on... Have you talked with any of the pipeline folks to make sure this will remain true over time, or if there there would be a better way to distinguish?

Reply:
@mccalluc There was a lot of back and forth a month or two ago with the pipeline folks and this was their suggestion.